web/src/main/java/org/akaza/openclinica/control/managestudy/UpdateStudyServletNew.java

```diff
@@ -649,4 +660,38 @@ private void updateInterviewerForSites(StudyBean studyBean, List<StudyBean> site
         }
 
     }
+    private String getPManageRegStatus(String studyOid) throws Exception
+    {
+		String pManageUrl = CoreResources.getField("portalURL");
```

Comment:
Looks like this code is duplicated in 2 different methods in the future please create one common method and reuse code. Also let's try to use RestTemplate going forward.